resources.go

```diff
@@ -146,3 +150,83 @@ func printRunGroups(api resourceAPI) error {
 	printResourceTable([]string{"Run Group ID", "Run Group Title"}, rows)
 	return nil
 }
+
+func postRunJUnitReport(c cliContext, runID int) error {
+	// Get the csv file path either and skip uploading if it's not present
+	fileName := c.String("junit-file")
+	if fileName == "" {
+		return nil
+	}
+	api = rainforest.NewClient(c.String("token"), c.Bool("debug"))
+
+	cmd := []string{
+		"rainforest-cli",
+		"report", strconv.Itoa(runID),
+		"--skip-update", // skip auto-updates for reports inside a run
+	}
```

Comment:
Why create and execute a new command rather than just calling `resource.GetRunJunit` here?

Reply:
I stated that way, but it needed more stuff brought in here to do than was nice. When I was fixing the reruns code's env, I thought that way kept it simpler.